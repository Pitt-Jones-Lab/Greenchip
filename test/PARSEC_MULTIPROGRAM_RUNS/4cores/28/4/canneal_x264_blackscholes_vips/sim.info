{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/4/canneal_x264_blackscholes_vips',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_4_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/4/canneal_x264_blackscholes_vips',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpmaFbNA/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb12',
 'pin_version': 'unknown',
 'rusage': (15636.246929000001,
            589.47638500000005,
            314608,
            0,
            0,
            0,
            77926,
            2,
            0,
            10744,
            1400,
            0,
            0,
            0,
            84032647,
            8143636),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/4/canneal_x264_blackscholes_vips --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem28.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_4_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpmaFbNA/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/4/canneal_x264_blackscholes_vips/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 6680.9424920082092,
 't_start': 1463065724.4341049,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1728450560L}
