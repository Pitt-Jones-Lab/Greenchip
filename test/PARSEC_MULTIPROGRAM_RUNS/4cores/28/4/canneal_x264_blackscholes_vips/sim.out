                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  1666500540 |  2335482449 |  2725528524 |  3271135120
  Cycles                           |  1468912116 |  1468912116 |  1468912116 |  1468912116
  IPC                              |        1.13 |        1.59 |        1.86 |        2.23
  Time (ns)                        |   552222600 |   552222600 |   552222600 |   552222600
  Idle time (ns)                   |       90417 |       50811 |      250200 |      964900
  Idle time (%)                    |        0.0% |        0.0% |        0.0% |        0.2%
Branch predictor stats             |             |             |             |            
  num correct                      |   163047861 |   128794024 |   324344530 |   290352561
  num incorrect                    |      200008 |     7239499 |     9662371 |     1788702
  misprediction rate               |       0.12% |       5.32% |       2.89% |       0.61%
  mpki                             |        0.12 |        3.10 |        3.55 |        0.55
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   216829047 |   265697639 |   374438880 |   401232792
    num misses                     |        1982 |       12953 |        6201 |       10349
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.01 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   540326962 |   758358154 |   788406470 |   927911489
    num misses                     |      205313 |     3061422 |      223803 |     3039580
    miss rate                      |       0.04% |       0.40% |       0.03% |       0.33%
    mpki                           |        0.12 |        1.31 |        0.08 |        0.93
  L2 TLB                           |             |             |             |            
    num accesses                   |      207295 |     3074375 |      230004 |     3049929
    num misses                     |      121145 |      424122 |       83247 |      371882
    miss rate                      |      58.44% |      13.80% |      36.19% |      12.19%
    mpki                           |        0.07 |        0.18 |        0.03 |        0.11
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   216829047 |   265697639 |   374438880 |   401232792
    num cache misses               |       74098 |    13419450 |      183727 |     5618074
    miss rate                      |       0.03% |       5.05% |       0.05% |       1.40%
    mpki                           |        0.04 |        5.75 |        0.07 |        1.72
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   540326962 |   758358154 |   788406470 |   927911489
    num cache misses               |     8252872 |    13887329 |     6872793 |    15610190
    miss rate                      |       1.53% |       1.83% |       0.87% |       1.68%
    mpki                           |        4.95 |        5.95 |        2.52 |        4.77
  Cache L2                         |             |             |             |            
    num cache accesses             |    12176118 |    28111732 |     8192685 |    22069139
    num cache misses               |     5786754 |     4828626 |     2687132 |     3174804
    miss rate                      |      47.53% |      17.18% |      32.80% |      14.39%
    mpki                           |        3.47 |        2.07 |        0.99 |        0.97
DRAM summary                       |             |             |             |            
  num dram accesses                |    29542001 |           0 |           0 |           0
  average dram access latency (ns) |      109.14 |         inf |         inf |         inf
  average dram queueing delay      |      109.14 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     4906786 |     3633958 |     2498965 |     2882272
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
